### PR TITLE
Spms 586

### PR DESCRIPTION
**Pull request by sanderPostma**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -29,13 +29,13 @@
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-        <java.version>1.8</java.version>
-        <spring.version>4.3.3.RELEASE</spring.version>
+        <java.version>11</java.version>
+        <sphereon-commons.version>1.1.0-SNAPSHOT</sphereon-commons.version>
```
Comment (nklomp) [inline pom.xml:33]: This is not open-source!

File: pom.xml [hunk 2]
```diff
@@ -86,24 +86,22 @@
...
             <dependency>
-                <groupId>junit</groupId>
-                <artifactId>junit</artifactId>
-                <version>${junit.version}</version>
-            </dependency>
-            <dependency>
-                <groupId>org.springframework</groupId>
-                <artifactId>spring-context</artifactId>
-                <version>${spring.version}</version>
+                <groupId>com.sphereon.ms</groupId>
```
Comment (nklomp) [inline pom.xml]: Not open-source!

**Review by nklomp — CHANGES_REQUESTED** [review]
You are including non open-source poms

File: pom.xml (hunk 2 above)
Comment (sanderPostma) [reply pom.xml]: That's fixed now, but the question is whether this project should be updated. 
The old Java 8 artifact runs perfectly fine with the newer ms/JVM. (As long as  you have a java.xml.bind dependency in your project)

Comment (nklomp) [reply pom.xml]: No it is a lib open to anybody so it makes sense to have the JVM level as low as possible

File: pom.xml (hunk 1 above)
Comment (nklomp) [reply pom.xml:33]: Move back to 1.8

**Pull request closed without merging**